index.html

```diff
@@ -0,0 +1,424 @@
+                        <strong>Jan 12,2022</strong>
+                    </td>
+                    <td rowspan="2">
+                        <em>$783.22</em>
+                    </td>
+                    <td rowspan="2">
+                        <button>
+                            <img src="img/points.png" alt="points">
+                        </button>
+                    </td>
+                    
+                </tr>
+                <tr>
+                    <td>
+                        <a href="https://www.wix.com/">https://www.wix.com/</a>
+                    </td>
+                    <td>
+                        <em>***** 2468</em>
+                    </td>
+                    <td>
+                        <a href="mailto:Itaibrahca31@gmail.com">Itaibrahca31@gmail.com</a>
+                    </td>
+                    <td>
+                        <em>$783.22</em>
+                    </td>
+                </tr>
+                <tr>
+                    <td rowspan="2">
+                        <input type="checkbox">
+                    </td>
+                    <td rowspan="2">
+                        <img src="img/youtube.png" alt="youtube">
+                    </td>
+                    <td>
+                        <a href="https://www.youtube.com/">Youtube</a>
+                    </td>
+                    <td>
+                        <strong>Visa</strong>
+                    </td>
+                    <td>
+                        <strong>Itai Bracha</strong>
+                    </td>
+                    <td>
+                        <strong>Jan 2,2022</strong>
+                    </td>
+                    <td rowspan="2">
+                        <img src="img/done.png" alt="done">
+                        <span>Done</span>
+                    </td>
+                    <td rowspan="2">
+                        <hstrong>Jan 12,2022</strong>
+                    </td>
+                    <td rowspan="2">
+                        <em>$783.22</em>
+                    </td>
+                    <td rowspan="2">
+                        <button>
+                            <img src="img/points.png" alt="points">
+                        </button>
+                    </td>
+                    
+                    <tr>
+                        <td>
+                            <a href="https://www.youtube.com/">https://www.youtube.com/</a>
+                        </td>
+                        <td>
+                            <em>***** 2468</em>
+                        </td>
+                        <td>
+                            <a href="mailto:Itaibrahca31@gmail.com">Itaibrahca31@gmail.com</a>
+                        </td>
+                        <td>
+                            <em>$783.22</em>
+                        </td>
+                    </tr>
+                </tr>
+            </tbody>
+        </table>
+        <div>
+            <div>
+                <button>

```

Comment:
![image](https://user-images.githubusercontent.com/104353194/167267995-060cd488-d030-4272-bc5b-998f232bea27.png)

![image](https://user-images.githubusercontent.com/104353194/167268030-a23e9afb-7578-4a48-ab40-7f6478ad8b32.png)

Reply:
you need to make common wrapper for the table and pagination div for further styles